README.md

```diff
@@ -25,6 +25,16 @@ We track our tasks and discuss their status openly using our GitHub project [Kan
 Typically, a card bundles tasks about one artifact (e.g. written articles accompanying one learning path section) or small milestones (e.g. finishing the post production of one section). 
 You can infer whether a task is currently actively developed or under review based on the card's column in the [Kanban board].
 
+## Trusted Committers
+
+These community members continually give their time to the running of this project.
+Special to:
```

Comment:
This read like there might have been a word missing.

```suggestion
Special thanks to:
```

Reply:
You’re right!   Thank you!